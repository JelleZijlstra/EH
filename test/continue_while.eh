private i = 0

while true
    i += 1
    echo i

    if i < 2
        continue
    end

    if i > 5
        break
    end

    echo "got to the end!"
end

for i in 6
    echo i

    if i < 2
        continue
    end

    if i > 5
        break
    end

    echo "got to the end!"
end
