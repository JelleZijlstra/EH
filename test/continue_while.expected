%%stdout%%
1
2
got to the end!
3
got to the end!
4
got to the end!
5
got to the end!
6
0
1
2
got to the end!
3
got to the end!
4
got to the end!
5
got to the end!
%%stderr%%
